Dockerfile

```
@@ -17,26 +17,33 @@ RUN apt-get update && \
 RUN add-apt-repository -y ppa:ondrej/php && \
     apt-get update
 
-# Install PHP 8.1 and required extensions
+# Install PHP 8.2 and required extensions
 RUN apt-get install -y \
```

Hadolint (2.14.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)